scripts/fetch-github-data.js

```diff
@@ -49,99 +100,101 @@ export const fetchGithubRateLimit = async () => {
   };
 };
 
-// const repoRequestCache = {};
-
-export const fetchGithubData = async data => {
+const getUpdatedUrl = async url => {
```

Comment:
Sometimes the graphql API doesn't pick up redirects. This function gets the updated url using the rest API to retry.